src/components/UncontrolledTable.js

```diff
@@ -20,7 +20,8 @@ export default class UncontrolledTable extends React.Component {
       ascending: PropTypes.bool
     }),
     onSelect: PropTypes.func,
-    onSort: PropTypes.func
+    onSort: PropTypes.func,
+    onVisibleRowsChanged: PropTypes.func
```

Comment:
I think this should just be `onVisibleRowsChange` to keep it consistent with react conversion, we also have an `onPageChange` prop couple lines up
```suggestion
    onVisibleRowsChange: PropTypes.func
```

Reply:
Good catch! Changing.